frame/support/test/tests/construct_runtime_ui/invalid_migration_param.rs

```diff
@@ -17,7 +17,8 @@ impl frame_system::Config for Runtime {
 struct Dummy;
 
 construct_runtime! {
-	pub struct Runtime<Dummy>
+	type Migrations = (Dummy);
```

Comment:
Does it work if you use `(Dummy,)`?

Reply:
Yes, it parses correctly and throws the correct error.